ldap3/protocol/rfc4512.py

```diff
@@ -446,7 +446,7 @@ def from_definition(cls, definitions):
                 else:
                     raise LDAPSchemaError('unknown schema definition class')
 
-                splitted = re.split('( NAME | DESC | OBSOLETE| X-| E-' + pattern + ')', object_definition[1:-1])
+                splitted = re.split('( NAME | DESC | OBSOLETE| X-| E-' + pattern + ')', str(object_definition[1:-1]))
```

Comment:
Yes, also because ``object_definition`` should be of "unicode" type. Please send me a test case with th error and the content of the object_definition variable.